packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -616,6 +614,9 @@ contract DeployImplementations is Script {
         });
         setters[2] = opcmSystemConfigSetter(_dii, _dio);
         setters[3] = OPContractsManager.ImplementationSetter({
+            // TODO: rename?
+            // This is used to set the name of the contract in the
+            // `OPContractsManager.implementations.releases` mapping. Can that change between releases?
             name: "OptimismMintableERC20Factory",
```

Comment:
I tried to rename this and it broke a bunch of stuff, didn't want to go down the whole rabbit hole of trying to fix it. I figure that the easiest way to handle a rename would be to allow for an OR in the naming for some time until we have upgraded all chains sufficiently past this point. It could also be more trouble than its worth doing the rename, the maintainers of OPCM should make this decision

Reply:
I went through exactly the same experience, made the change then reverted it. Technically it should be OK to rename contracts between releases, but the number of things that broke was quite surprising so I decided just to revert it. It's fine as is for now.